build.go

```diff
@@ -241,6 +231,20 @@ func main() {
 	}
 }
 
+func fixGoPath() {
+	// If GOPATH isn't set, set it correctly with the assumption that we are
+	// in $GOPATH/src/github.com/syncthing/syncthing.
+	if os.Getenv("GOPATH") == "" {
```

Comment:
I think we should adhere to the standard Go way of documenting things, which is putting the comment above the function definition starting with `fixGoPath does X Y Z`